x-pack/test/security_solution_api_integration/test_suites/genai/knowledge_base/entries/utils/remove_server_generated_properties.ts

```diff
@@ -29,8 +29,10 @@ export type EntryWithoutServerGeneratedProperties = Omit<
 export const removeServerGeneratedProperties = (
   entry: KnowledgeBaseEntryCreateProps
 ): EntryWithoutServerGeneratedProperties => {
-  const removedProperties = omit(entry, serverGeneratedProperties);
-
+  const removedProperties = {
+    ...omit(entry, serverGeneratedProperties),
+    users: map(entry.users, (user) => omit(user, 'id')),
+  };
```

Comment:
Is this also related to changes from https://github.com/elastic/kibana/pull/211367? Why scrub the `id`?

Reply:
this is test utils, we don't have `profile_id` and we just want to make sure that the proper `username` was added to the entry